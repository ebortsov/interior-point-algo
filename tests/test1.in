4 2
1 1 0 0
2 4 1 0
1 3 0 -1
16 9
0.5 3.5 1 2
0.00001