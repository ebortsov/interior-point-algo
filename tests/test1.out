alpha = 0.50: [5.9999938 1.0000027 0.0000018 0.0000018]
alpha = 0.90: [5.9999978 1.0000009 0.000001  0.0000004]
